### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henry-richard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,3 +32,4 @@ Test4
 
 Test 4
 test 1 - pullflow{aa}
+test2.2 - draft to ready for review
```
Comment (henry-richard) [inline README.md:35]: Pullflow Icon

**Comment by henry-richard** [comment]
**Slack user** please see the comment

File: README.md [hunk 2]
```diff
@@ -32,3 +32,4 @@ Test4
 
 Test 4
 test 1 - pullflow{aa}
```
Comment (chad-daniels) [inline README.md:34]: remove this

Comment (chad-daniels) [inline README.md:34]: testing review

**Pull request merged**